definitions/npm/underscore.string_v3.x.x/flow_all/underscore.string_v3.x.x.js

```diff
@@ -0,0 +1,167 @@
+    isBlank(): boolean;
+    join(...strings: Array<string>): Chain,
+    lines(): Array<string>;
+    wrap(options: $npm$wrap$options): Chain;
+    dedent(pattern?: string): Chain;
+    reverse(): Chain;
+    splice(index: number, howmany: number, substring: string): Chain;
+    startsWith(starts: string, position?: number): boolean;
+    endsWith(ends: string, position?: number): boolean;
+    pred(): Chain;
+    succ(): Chain;
+    titleize(): Chain;
+    camelize(decapitalize?: boolean): Chain;
+    camelcase(decapitalize?: boolean): Chain, // Alia;
+    classify(): Chain;
+    underscored(): Chain;
+    dasherize(): Chain;
+    humanize(): Chain;
+    trim(characters?: string): Chain;
+    strip(characters?: string): Chain, // Alia;
+    ltrim(characters?: string): Chain;
+    lstrip(characters?: string): Chain, // Alia;
+    rtrim(characters?: string): Chain;
+    rstrip(characters?: string): Chain, // Alia;
+    truncate(length: number, truncateString?: string): Chain;
+    prune(length: number, pruneString: string): Chain;
+    words(delimiter: string|RegExp): Array<string>;
+    sprintf(...arguments: Array<*>): Chain;
+    pad(length: number, padStr?: string, type?: $npm$pad$type): Chain;
+    lpad(length: number, padStr?: string): Chain;
+    rjust(length: number, padStr?: string): Chain, // Alia;
+    rpad(length: number, padStr?: string): Chain;
+    ljust(length: number, padStr?: string): Chain, // Alia;
+    lrpad(length: number, padStr?: string): Chain;
+    center(length: number, padStr?: string): Chain, // Alia;
+    toNumber(decimals?: number): number;
+    strRight(pattern: string): Chain;
+    strRightBack(pattern: string): Chain;
+    strLeft(pattern: string): Chain;
+    strLeftBack(pattern: string): Chain;
+    stripTags(): Chain;
+    // --
+    repeat(count: number, separator?: string): Chain,
+    surround(wrap: string): Chain,
+    quote(quoteChar?: string): Chain,
+    q(quoteChar?: string): Chain, // Alias
+    unquote(quoteChar?: string): Chain,
+    slugify(): Chain,
+    naturalCmp(string2: string): number,
+    toBoolean(truthy?: Array<string|RegExp>, falsy?: Array<string|RegExp>): ?boolean,
+    toBool(truthy?: Array<string|RegExp>, falsy?: Array<string|RegExp>): ?boolean, // Alias
+    map(iteratee: (character: string) => string): Chain,
+
+    // native Javascript string methods
+    toUpperCase(): Chain,
+    toLowerCase(): Chain,
+    split(separator?: string, limit?: number): Array<string>,
+    replace(pattern: string|RegExp, replacement: string|((...params: Array<string|number>) => string)): Chain,
+    slice(beginSlice: number, endSlice?: number): Chain,
+    substring(indexStart: number, indexStart?: number): Chain,
+    substr(start: number, length?: number): Chain,
+    concat(...params: Array<string>): Chain,
+
+    // Chain specific methods
+    tap(fn: (string: string) => string): Chain;
+    value(): string;
+  }
+
+  declare var exports: {
```

Comment:
@thejameskyle initially I used `declare module.exports` but it doesn't work for flow version prior to 0.25 😞.

Ok, moving `flow_all` to `flow_>=v0.25.x` then.